RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.cpp

```diff
@@ -17,6 +17,7 @@
 #include "detectorSegmentations/FCCSWGridModuleThetaMerged_k4geo.h"
 #include "detectorSegmentations/FCCSWGridPhiTheta_k4geo.h"
 #include "detectorSegmentations/FCCSWEndcapTurbine_k4geo.h"
+#include "detectorSegmentations/FCCSWHCalPhiTheta_k4geo.h"
```

Comment:
does this mean that one cannot use the phi-row segmentation for the fixed-size SW clustering? could this be added?

Reply:
Now it is added.